code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -386,8 +387,8 @@
 		/obj/item/assembly/flash/cyborg,
 		/obj/item/screwdriver/cyborg,
 		/obj/item/crowbar/cyborg,
-		/obj/item/stack/tile/plasteel/cyborg,
-		/obj/item/soap/nanotrasen,
+		/obj/item/harmalarm,
+		/obj/item/reagent_containers/borghypo/peace,

```

Comment:
I hope this isn't intentional

Reply:
:^)

Reply:
Are you *really* sure you wanna remove the cleaning tools from the cleaning cyborg?

Reply:
wait what